projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -118,6 +118,10 @@ function ( $track ) {
 				$tracks = $this->get_track_list();
 			}
 
+			if ( is_wp_error( $tracks ) ) {
+				return $tracks;
```

Comment:
Couldn't we return a `WP_Error` here as well?

Reply:
I had imagined a custom error message, yes, but maybe that's just me. :) I'm not insisting on this!

Reply:
Yes, I thought about that too but looking more closely into the potential WP_Errors returned by [load_feed](https://github.com/Automattic/jetpack/blob/11d94d0ff464a897e1335de572335ec8a9e44959/projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php#L358), it looks like it already handles returning a single error in case [core's](https://developer.wordpress.org/reference/functions/fetch_feed/) `fetch_feed` returns a `WP_Error`